src/gui/qgsannotationitem.cpp

```diff
@@ -392,6 +395,7 @@ void QgsAnnotationItem::_writeXML( QDomDocument& doc, QDomElement& itemElem ) co
   annotationElem.setAttribute( "mapPositionFixed", mMapPositionFixed );
   annotationElem.setAttribute( "mapPosX", qgsDoubleToString( mMapPosition.x() ) );
   annotationElem.setAttribute( "mapPosY", qgsDoubleToString( mMapPosition.y() ) );
+  annotationElem.setAttribute( "mapPosCrsWkt", mMapPositionCrs.toWkt() );
```

Comment:
```.toProj4()``` could also be used here. Would that be better? It certainly produces much less text than the WKT.